### DISCUSSION TITLE
Add more open shell start menu skins.

### DISCUSSION THREAD

**Discussion by Techfromthepast123** [Open-Shell/Open-Shell-Menu#1224] [Ideas]
Can we add theses skins to open shell ?
Windows 7:Aero,Basic and Classic theme.
Windows vista:Aero,basic and classic.
Windows XP:Royale,Royale noir,Zune,Classic and Embedded styles.
Also for the windows XP style can you change the style of the shutdown button to the XP style one ?(The one with the bar at the bottom with the shutdown button and logoff button next to it without any arrow.)
And also add the windows vista style one (the windows 7 style but has a shutdown button next to a lock button and they are icons not texts.)
And can you add an option to change the windows update icon style ? With preinstalled ones?
The windows 7 blue windows update icon.
The windows 7 orange exclamation mark shield.
And custom style image.
Thanks!!!

**Comment by blackcrack** [comment]
if you have anyone who be in able for build some skins or you.. any can make a patch and skins and add it via patch, so if you in able for make some  patches for add some Skins, i guess the team is happy to able some things to add to the packages..
would be nice if it's happens..

**Comment by Techfromthepast123** [comment]
I don't know how to build Open shell skins.
And Open shell does'nt have a feature to have windows vista style shutdown button.

**Reply by VikoVikoVikoVikoVikoVikoViko** [reply]
Yes it does.

**Comment by blackcrack** [comment]
[Classic+Open-Shell_Almanach.odt](https://github.com/Open-Shell/Open-Shell-Menu/files/9843804/Classic%2BOpen-Shell_Almanach.odt)

**Comment by ijustatemyTV** [comment]
plz add all the windows XP themes to the windows 7 layout themes
:D
